pkg/endpointmanager/conntrack.go

```diff
@@ -67,6 +65,8 @@ func runGC(e *endpoint.Endpoint, ipv4, ipv6 bool, filter *ctmap.GCFilter) {
 		}
 		defer m.Close()
 
+		mapType = m.MapInfo.MapType
```

Comment:
Is it at all possible that the map types differ between each map stored in `maps`?

Reply:
Not sure but it seems like the safest option to always use the latest map type available